From 78371fb92709107ed252eeb194ee922651877fc7 Mon Sep 17 00:00:00 2001
From: Jiyong Park <jiyong@google.com>
Date: Wed, 6 Sep 2017 12:10:07 +0900
Subject: [PATCH 09/11] msm8974: Don't rely on transitively included headers
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

One must explicitly include what it need.
time.h for struct tm.
sys/time.h for gettimeofday, etc.

Bug: 37629934
Test: build

Merged-In: Iacd5317d958f61df062fca40fbf2e8992c7fbb44
Change-Id: I19e1043f23dbe85836b73714a4f1440159ebe419
Keep msm8974 in sync with the rest, post removal
Signed-off-by: D. Andrei Măceș <Andrei.Maces@alumni.nd.edu>
---
 msm8974/utils/loc_log.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/msm8974/utils/loc_log.cpp b/msm8974/utils/loc_log.cpp
index 392966e..f961631 100644
--- a/msm8974/utils/loc_log.cpp
+++ b/msm8974/utils/loc_log.cpp
@@ -32,6 +32,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/time.h>
+#include <time.h>
 #include "loc_log.h"
 #include "msg_q.h"
 #ifdef USE_GLIB
-- 
2.17.1

